README.md

```diff
@@ -108,6 +108,7 @@ A configuration wizard will prompt you to enter the necessary configuration para
   - call - OTP via Voice call
   - sms - OTP via SMS message
 - resolve_aws_alias - y or n. If yes, gimme-aws-creds will try to resolve AWS account ids with respective alias names (default: n). This option can also be set interactively in the command line using `-r` or `--resolve` parameter
+- include_path - y or n. This is optional. If yes the full role path to the role name will be included in your profile name. (default: n)
```

Comment:
Could you provide an example here, for the sake of the documentation, of the difference between `y` and `n` settings?

Reply:
yes, I will append something like this: 

If `y`: `<acct>-/some/path/administrator`. If `n`: `<acct>-administrator`